run/faasm.py

```diff
@@ -1,104 +0,0 @@
-#!/usr/bin/python3
-import json
-import math
-import re
-import sys
-import time
-
-from os import environ
-from os.path import dirname, join, realpath
-from subprocess import check_output
-
-from faasmcli.util.call import invoke_impl
-
-PROJ_ROOT = dirname(dirname(realpath(__file__)))
-BASE_DIR = "{}/../..".format(PROJ_ROOT)
-LAMMPS_USR = "lammps"
-LAMMPS_FUNC = "main"
-LAMMPS_CMDLINE = "-in faasm://lammps-data/in.controller"
-
-# Benchmark details
-# TODO decide where common config may live
-NUM_PROCS = [1, 2, 4]
-NUM_TESTS = 1
-
-
-# Update k8s host and port env variables
-def update_env():
```

Comment:
As mentioned elsewhere, we can just assume the default `kubectl` command is set up, and run from outside the containers for now, so this is unnecessary.